app/src/main/java/com/example/event_reminder/SplashActivity.kt

```diff
@@ -1,26 +1,43 @@
 package com.example.event_reminder
 
+import android.annotation.SuppressLint
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.os.Handler
 import android.view.WindowManager
+import java.util.*
+import java.util.TimerTask
+
+
+
+
 
 class SplashActivity : AppCompatActivity() {
+    //Timer for starting MainActivity
+    var timer = Timer()
```

Comment:
No need for a timer as well,
Ref : https://www.androidhive.info/2013/07/how-to-implement-android-splash-screen-2/